ethereum/nft/tokens.sql

```diff
@@ -538,6 +538,4284 @@ COPY nft.tokens (contract_address, name, symbol, standard) FROM stdin;
+\\x6fc355d4e0ee44b292e50878f49798ff755a5bbc	DeadHeads	DEAD	ERC721
+\\xdf3e1410841124603c89e2a7823bb6a34dda4ee0	FYDCards	FYDCards	ERC1155
+\\x15bddefa4a610699defcd858a09a3071f744b938	The Avatar Project	TAP	ERC721
+\\xb1b0dbd358dd688d803784b467c2584d3b2f4094	OFFBLUE	OBLUE	ERC1155
+\\x0fb69d1dc9954a7f60e83023916f2551e24f52fc	LostSoulsSanctuary	LSS	ERC721
+\\x8186a255354f9e3c2aea35e271d8df9a6cdc1b7c	Grunge Dreams	H/\L0	ERC721
+\\xe9b4ad578264d16b55eb25e6eec1999306d912bd	Real World Ceremony Invite in the Virtual World		ERC721
+\\x3ec057be60db3a2d416e14a6e3ab7f37ffccd7b8	GenSys.X	GenSys.X	ERC1155
+\\x773f36b9cada2bed56b7b746b0cba7490c0333d5	Cold Truth Culture UNIFTY Collection	CTC	ERC1155
+\\x40e974c15210284b966e44e3483cf3b8ed558490	Springfield Blockchain 	SPRING	ERC1155
+\\x60e795f7aa87c0def180a9754e60d21325b63ec6	Avostrology	AVO	ERC1155
+\\xe68e16178f51f84d60863fd7f06bcd1cbbc853a9	FuturamaCryptoCans	FCT	ERC721
+\\x9a40ceb075d5ecb43de0a399a9b4ace110fdbdb2	Stamps	STAMP	ERC721
+\\x676e18a3a73c7185041f901a0da4dfbf670e04d6	Icons Open Edition by Klarens Malluta	ICONSOPENEDITIONBYKLARENSMALLUTA	ERC721
+\\x5f5ff74b198e5095069cf132d2523dbae516613f	MonsterBit	MB	ERC721
+\\xfe9231f0e6753a8412a00ec1f0028a24d5220ba9	Unidentified contract		ERC1155
+\\xd448e6cca10ff5d1ce52ddc6b6fc4bfcb796d8eb	CashGrabNFT	CGNFT	ERC721
+\\x600a4446094c341693c415e6743567b9bfc8a4a8	Anchor Certificates	ACS	ERC721
+\\x8830eba9a50a470cd0cf28aafd7255f37345909a	Official Tiger King NFT 	Home of Joe Exotic 	ERC721
+\\xde6123d1e7b0483b16e745ac329750b170ef79d8	The Title Open Edition by Pak	THETITLEOPENEDITIONBYPAK	ERC721
+\\x13362c05d9bfc5a0e9803db1122d4329294e97a9	Rusla		ERC721
+\\x267129c5dfddbbc3fa2da7f69e69e6f81736a21a	Empress	NUDE	ERC721
+\\x6184f10302cebeea0211f9310225f051cc549626	Galactic Secret Agency	GSA	ERC721
+\\xa0a6093e4d3ce245e1849231f396e8b20d3ba341	Myōbu Silver Cards	MYOBUAG	ERC1155
+\\x121a03f7b2427cc5546f4c9bf6f2dd43a36b3d01	infectedpokémon	INPOKE	ERC1155
+\\xadc97941cea7e33c436917d6dd8f1afb69ecb36b	Fly me to the moon by Dot Pigeon	FLYMETOTHEMOONBYDOTPIGEON	ERC721
+\\x32aa07e6ebf2340eef1f717197ade6982b815536	Unidentified contract		ERC1155
+\\x9fd5c67862aaae14a4c4a44fd16f9afe507649de	TS Moreau Sketches		ERC721
+\\x3e30ca44cd35169bfd98e8ef2c80f06e4c97f6f1	TEMPLATE	TMPLT	ERC721
+\\x41a2788b3599ae8f8fe5da4055e4ad89fdf33ee3	Lite Studios 		ERC721
+\\xa5f4858ada386758e62165037af3fe21655068de	NIFTY HOROSCOPE		ERC721
+\\x5c70591acaf87ddee68b50877d4975e498a32c1f	Unidentified contract		ERC721
+\\x1ae6593db3e5fecb9d1ef5c4ebc81eb46999d6e8	No Labels	NOLABELS	ERC1155
+\\x6caa272b1e9a074a66d82e7f77437e3ba5cdbcd0	Slickmau5 by deadmau5 x OG Slick	SLICKMAUBYDEADMAUXOGSLICK	ERC721
+\\x56b75789580afcbce98db2f16c2cc8d037147da1	The Goose's Gander	by BruceTheGoose	ERC721
+\\x4624573b7a782b3112b23424683f2afe7aed05fb	SUPER PUMP (or BITCOIN - A STRANGE CRITTER)	SUPERPUMP1	ERC721
+\\x3163d2cfee3183f9874e2869942cc62649eeb004	DCL DCG		ERC721
+\\xe8
```

Comment:
Are these emojis gonna work?

Reply:
I understood from @va3093 that unicode and emojis should work.